### ISSUE TITLE
Facehugger issues 2 electric boogaloo

### ISSUE THREAD

**Issue by CptWad** [d3athrow/vgstation13#7073]
I opened this issue about 7 months (#4269) ago and it was lost in time
Reopening because I actually watched the very same shit happen all over again in a round

**//FOLLOWING IS OLD READ FARTHER DOWN THE THREAD\\\**

1. They do not jump on anyone with something in their mask slot. This includes (Tested) gas masks and other (dead) facehuggers.
2. Facehuggers leave a sprite of themselves on the floor when jumping on someone
3. Facehuggers do not provide oxygen(or proper air now that vox and plasman exist) anymore. (They used to, that way when dumbshit mchunter breaks a window the monkeys did not all die)
4. Facehugger AI generally wonky. In game to get people face hugged I had to drag them over an active face hugger without them having a mask or other xeno-protection gear
5. Thrown facehuggers do not land of faces. In fact throwing them is a lot like an AOE banana peel rather than a face full of xeno

**Comment by gbasood** [comment]
Weird, just last week facehuggers were actually ripping other facehuggesr off the same face repeatedly back and forth, and now we get that first problem

**Comment by Ephx1** [comment]
Facehuggers need some serious overhauls, right now they're incredibly shitty buggy masks and I cringe every time I see them. It seems like a new update switches their bugs around too.

**Comment by CptWad** [comment]
I recently put facehuggers forcefully onto monkeys faces
they did not get infested
I had to drag them over facehuggers in order for the facehuggers to jump onto the face THEN impregnate

I also had an idea for facehuggers: Give them cockroach AI and similarities. Have them crawl on the walls and rush towards humans w/o masks and shit

**Comment by CptWad** [comment]
I retested the bugs
1. Facehuggers WILL NOT JUMP ONTO SOMEONES FACE IF THERE IS A MASK ON THAT FACE
1a. Tested with a regular gas mask, the one found in an internals box. They had no other clothes on
2. When forcibly putting a facehugger on a monkey it does not infest them
3. Throwing facehuggers seems suddenly fixed. It also goes through masks properly.
4. Currently still, the best way to facehug people is to drag them over facehuggers
